client/src/views/Profile/Card/switchVideo.vue

```diff
@@ -4,6 +4,9 @@
       <q-tab name="my-videos">
         <i class="far fa-play-circle"></i>
       </q-tab>
+      <q-tab name="liked-videos">
```

Comment:
<q-tab> and <q-tab-panel> could go with v-for now (aka computed array of objects, less templating and more readability)